.travis.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     include:
     -   stage: Tests
         name: "Smoke, Functional and Acceptance Tests"
-        script: docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev-quick error-pages-generate checks-ci
+        script: docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev-quick backend-api-install backend-api-oauth-keys-generate error-pages-generate checks-ci
```

Comment:
I think that [the Travis build fails](https://travis-ci.org/shopsys/shopsys/jobs/553287934) because you first run `build-demo-dev-quick` which executes DB migrations without the API bundle, then it installs the API bundle and does not run the additional migrations. Maybe add `db-migrations` before the `checks-ci`? Maybe `clean-redis` will be needed as well as the Doctrine metadata might get cached...

Not sure why it failed now though.